commerce-inventory-service/src/main/java/com/liferay/commerce/inventory/service/impl/CommerceInventoryWarehouseItemLocalServiceImpl.java

```diff
@@ -54,6 +54,34 @@ public CommerceInventoryWarehouseItem addCommerceInventoryWarehouseItem(
 			commerceInventoryWarehouseItem);
 	}
 
+	@Override
+	public CommerceInventoryWarehouseItem addCommerceInventoryWarehouseItem(
```

Comment:
What would you think of replacing the old `addCommerceInventoryWarehouseItem` implementation with a call to this new method? Looks like all the params are the same except for the addition of `externalReferenceCode` -- might be nice to simplify.